content/blog/2016/2016-03-15-jenkins-certification.adoc

```diff
@@ -0,0 +1,33 @@
+In the IT world, namely in software, “certification” is used in many different ways and for many different purposes, from very simple and light certifications to very heavy and complex ones. In the “light” category, you can usually be certified on the basis of a short quiz at the end of an online training. At the other end of the spectrum, certifications are based on a proctored multiple-choice questionnaire-based exam and/or hands-on labs. In some industries, certifications are even more demanding. For instance, to become a Certified Public Accountant in the USA, you have to pass a standard examination and, on top of this, each state/jurisdiction has its own set of education and experience requirements that individuals must meet.
+
+== What Type of Certification for Jenkins and for What Purpose?
+When we started our internal discussions at CloudBees regarding a certification program for Jenkins, we were aware of this broad set of certification definitions. Therefore, our first goal was to define what type of certification we wanted to develop and for what purpose. We quickly agreed on the fact that it should be _a professional-grade certification, whose purpose would be to provide a professional standard for the Jenkins ecosystem,_ benefiting both individuals and organizations, thanks to a common, respected and well-known body of knowledge and practice. “Professional” means that you have the expected level of skills and experience in order to leverage them in a professional environment, in enterprise projects or as a consultant for instance. +
+
+Many members of the CloudBees management team have firsthand experience with certification programs developed in other IT ecosystems such as telecoms (Cisco), infrastructure (Microsoft, Red Hat) or business applications (SAP), to name a few. This was definitely the type of professional certification we wanted to bring to Jenkins. We knew it would represent a substantial investment, but we also knew that the whole Jenkins ecosystem would benefit, whether at the overall community or individual level, as well as IT organizations, system integrators or recruiting firms looking for qualified Jenkins personnel.
+
+== How Was The Certification Built?
+I have had the privilege to supervise the creation and implementation of the Jenkins Certification Program at CloudBees. The program is comprised of two certifications: Certified Jenkins Engineer (CJE) for Jenkins certification, and Certified CloudBees Jenkins Platform Engineer (CCJPE) for  certification on the CloudBees Jenkins Platform. +
+
+We started by creating a Certification Advisory Board whose members are: Kohsuke Kawaguchi, Jenkins creator and CTO at CloudBees; Harpreet Singh, VP products at CloudBees; Oliver Gondza, initially representing the Jenkins community; Jason Shawn, senior director DevOps at Ellucian, representing the CloudBees customer constituency; and Jose Alvarez, managing director at Zivra, representing the CloudBees partner ecosystem. +
```

Comment:
"initially representing the Jenkins community" seems a bit incorrect to me since nobody voted for making @olivergondza as such representative AFAIK && he is not a member of Jenkins board, which represents the community at the official level. I suppose he formally represented himself, right?